### PR TITLE
feat: [CDS-45477]: change env type as BE is changed

### PR DESCRIPTION
**Pull request by ishantmehtaharness**
coverage-coverage is failing

Screenshot 2022-10-26 at 3 41 12 PM

### CODE REVIEW

**Review by kapilkumawat86 — DISMISSED** [review]

**Review by pankajsingh-harness — APPROVED** [review]

**Review by pankajsingh-harness — APPROVED** [review]

File: src/modules/72-freeze-windows/types.ts [hunk 1]
```diff
@@ -49,8 +49,8 @@ export interface ValidationErrorType {
 
 export enum EnvironmentType {
   All = 'All',
-  PROD = 'PROD',
-  NON_PROD = 'NON_PROD'
+  PROD = 'Production',
```
Comment (vkbansal-harness) [inline src/modules/72-freeze-windows/types.ts:52]:
```suggestion
  Production = 'Production',
```

File: src/modules/72-freeze-windows/types.ts [hunk 2]
```diff
@@ -49,8 +49,8 @@ export interface ValidationErrorType {
...
+  NON_PROD = 'PreProduction'
```
Comment (vkbansal-harness) [inline src/modules/72-freeze-windows/types.ts:53]:
```suggestion
  PreProduction = 'PreProduction'
```

File: src/modules/72-freeze-windows/types.ts (hunk 1 above)
Comment (ishantmehtaharness) [reply src/modules/72-freeze-windows/types.ts:52]: I thought I will keep the keys same

Comment (vkbansal-harness) [reply src/modules/72-freeze-windows/types.ts:52]: Enums should same

**Review by Eric761 — APPROVED** [review]

File: src/modules/72-freeze-windows/types.ts (hunk 1 above)
Comment (ishantmehtaharness) [reply src/modules/72-freeze-windows/types.ts]: done, updated

File: src/modules/72-freeze-windows/types.ts (hunk 2 above)
Comment (ishantmehtaharness) [reply src/modules/72-freeze-windows/types.ts]: done

**Review by vkbansal-harness — APPROVED** [review]

**Pull request merged**